app/styles/ui/_pull-request-badge.scss

```diff
@@ -0,0 +1,41 @@
+.toolbar-dropdown.open .pr-badge {
+  background: $gray-300;
+}
+
+.toolbar-dropdown:not(.open) .pr-badge {
+  background: $gray-700;
+}
+
+.pr-badge {
+  display: flex;
+  flex-direction: row;
+  height: 18px;
+
+  align-items: center;
+
+  border: var(--base-border);
+  border-radius: var(--border-radius);
+  border-width: 0;
+
+  margin-right: var(--spacing);
+  margin-left: var(--spacing);
```

Comment:
The text container already has a 10px right margin, so with this we're getting 20px. Can we ditch this?